docusaurus.config.js

```diff
@@ -215,6 +220,8 @@ const config = {
             to: '/organizations',
           },
         ],
+        // @ts-ignore
```

Comment:
instead of ts-ignore, please use `ts-expect-error`: it forces you to provide an reason why we can ignore it

Reply:
A good practice when you insert a `@ts-expect-error` is to write a comment about why you are ignoring it, something like:
```@ts-expect-error provided types doesn't match the real possible configuration```

This helps maintainers that will come after you to better understand what's going on.

Reply:
unresolving this because hasn't been resolved yet.